src/js/services/meta.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+module.exports = function(app) {
+  app.factory(
+    'metaService',
+    [
+      function() {
+        var data = {
+          title: undefined,
+          author: undefined,
+          description: undefined,
+          keywords: undefined
+        };
```

Comment:
We could add a function for setting the values:
```
data.set = function(newMeta) {
  angular.copy(newMeta, data);
};
```
If you only want to set the title (and set everything else to undefined) you would simply call
`metaService.set({title: 'PaperHub rocks'})` instead of setting each key individually.